src/shared/dbus-client.h

```diff
@@ -236,4 +236,12 @@ dbus_bool_t bluealsa_dbus_message_iter_get_pcm_props(
 		DBusError *error,
 		struct ba_pcm *pcm);
 
+dbus_bool_t bluealsa_dbus_pcm_select_codec(
+		struct ba_dbus_ctx *ctx,
+		const struct ba_pcm *pcm,
```

Comment:
Change this parameter to `const char *pcm_path` and place the definition just after `bluealsa_dbus_open_pcm`. The same thing in the C file.